mmcv/ops/nms.py

```diff
@@ -39,6 +39,37 @@ def symbolic(g, bboxes, scores, iou_threshold, offset):
             1)
 
 
+class SoftNMSop(torch.autograd.Function):
+
+    @staticmethod
+    def forward(ctx, boxes, scores, iou_threshold, sigma, min_score, method,
+                offset):
+        dets, inds = ext_module.softnms(
+            boxes.cpu(),
+            scores.cpu(),
+            iou_threshold=float(iou_threshold),
+            sigma=float(sigma),
+            min_score=float(min_score),
+            method=int(method),
+            offset=int(offset))
+        return dets, inds
+
+    @staticmethod
+    def symbolic(g, boxes, scores, iou_threshold, sigma, min_score, method,
+                 offset):
+        nms_out = g.op(
```

Comment:
Here we could add assertation that it should be exported to ONNX with Pytorch>=1.7.0.